### ISSUE TITLE
Feature: Add postgresql

### ISSUE THREAD

**Issue by yaronsumel** [digota/digota#2]
I got requests to add postgresql as optional storage handler.. any other interest in that ?

**Comment by sebastianmacias** [comment]
Definitely! digota looks very interesting and I will be soon evaluating it for a project, however postgresql support is a must in my case.

**Comment by yaronsumel** [comment]
K! one of the key points here is to be flexible as possible.. so it is going to happen soon with other storage handlers as well.

**Comment by ds0nt** [comment]
I'd also like postgres support. Would be useful because it could then run on google's GKE and cloud-sql

**Comment by goodforever** [comment]
The postgresql is my first selection
